debian/linglong-bin.postinst

```diff
@@ -1,98 +1,101 @@
+    deb-systemd-helper unmask 'org.deepin.linglong.PackageManager.service' >/dev/null || true
 
-        # was-enabled defaults to true, so new installations run enable.
-        if deb-systemd-helper --quiet was-enabled 'org.deepin.linglong.PackageManager.service'; then
-                # Enables the unit on first installation, creates new
-                # symlinks on upgrades if the unit file has changed.
-                deb-systemd-helper enable 'org.deepin.linglong.PackageManager.service' >/dev/null || true
-        else
-                # Update the statefile to add new symlinks (if any), which need to be
-                # cleaned up on purge. Also remove old symlinks.
-                deb-systemd-helper update-state 'org.deepin.linglong.PackageManager.service' >/dev/null || true
-        fi
+    # was-enabled defaults to true, so new installations run enable.
+    if deb-systemd-helper --quiet was-enabled 'org.deepin.linglong.PackageManager.service'; then
+        # Enables the unit on first installation, creates new
+        # symlinks on upgrades if the unit file has changed.
+        deb-systemd-helper enable 'org.deepin.linglong.PackageManager.service' >/dev/null || true
+    else
+        # Update the statefile to add new symlinks (if any), which need to be
+        # cleaned up on purge. Also remove old symlinks.
+        deb-systemd-helper update-state 'org.deepin.linglong.PackageManager.service' >/dev/null || true
+    fi
 fi
 
+# Add logging to systemctl command
 if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-deconfigure" ] || [ "$1" = "abort-remove" ] ; then
-        if [ -d /run/systemd/system ]; then
-                systemctl --system daemon-reload >/dev/null || true
-                if [ -n "$2" ]; then
-                        _dh_action=restart
-                else
-                        _dh_action=start
-                fi
-                deb-systemd-invoke $_dh_action 'org.deepin.linglong.PackageManager.service' >/dev/null || true
+    if [ -d /run/systemd/system ]; then
+        systemctl --system daemon-reload >/dev/null || true | tee -a "$log_file"
+        if [ -n "$2" ]; then
+            _dh_action=restart
+        else
+            _dh_action=start
         fi
+        deb-systemd-invoke $_dh_action 'org.deepin.linglong.PackageManager.service' >/dev/null || true | tee -a "$log_file" 
+    fi
 fi
-# End automatically added section
 
+# End automatically added section
 case "$1" in
 configure)
-        version=$2
-        shift
-        # enable kernel.unprivileged_userns_clone
-        if [ -f /etc/sysctl.d/linglong.conf ];then
-                sysctl -p /etc/sysctl.d/linglong.conf
-        fi
-        ;;
+    # enable kernel.unprivileged_userns_clone
+    sysctl_config="/etc/sysctl.d/linglong.conf"
+    if [ -f "$sysctl_config" ]; then
+        echo "Enabling kernel.unprivileged_userns_clone..." | tee -a "$log_file"
```

Comment:
Good job. We need this log.

Reply:
thanks, ill be learning to contribute to linglong, anything else or any specific task you can just tell me.